src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -107,10 +109,113 @@ public void Start()
                     UseShellExecute = false,
                     FileName = ExePath,
                     Arguments = GetCmdArgs(port),
-                    CreateNoWindow = true
+                    CreateNoWindow = true,
+                    WindowStyle = ProcessWindowStyle.Hidden,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true
+                }
+            };
+            process.OutputDataReceived += (sender, args) =>
+            {
+                string data = args.Data;
+                if (data != null)
+                {
+                    if (data.Contains("Server started"))
+                    {
+                        CreateChannelAndStremLogs();
+                    }
+                    if (IsVerbose())
+                    {
+                        WritelnToPane(data);
+                    }
+                }
+            };
+            process.ErrorDataReceived += (sender, args) =>
+            {
+                string data = args.Data;
+                if (data != null)
+                {
+                    WritelnToPane(data);
                 }
             };
-            process.Start();
+            if (IsVerbose())
+            {
+                WritelnToPane($"Running {ExePath}");
+            }
+            try
+            {
+                process.Start();
+                process.BeginOutputReadLine();
+                process.BeginErrorReadLine();
+            }
+            catch (Exception e)
+            {
+                Debug.WriteLine("Unable to start SonarLint daemon: {0}", e);
+                WritelnToPane($"Unable to start SonarLint daemon {e.Message}");
+            }
+        }
+
+        private void CreateChannelAndStremLogs()
+        {
+            channel = new Channel($"{DAEMON_HOST}:{port}", ChannelCredentials.Insecure);
+            daemonClient = new StandaloneSonarLint.StandaloneSonarLintClient(channel);
+            ListenForLogs();
+            StartHeartBeat();
```

Comment:
I really intend to keep them running in the background. I don't want to block on them.